en/frames_rover/aion_r1.md

```diff
@@ -0,0 +1,78 @@
+# Aion Robotics R1 UGV
+
+<Badge type="warning" text="main (PX4 v1.15)" />
```

Comment:
Added badge to indicate this is new in v1.15. I assume that for now this only works in v1.15?

Can you add a release note for this in https://docs.px4.io/main/en/releases/main.html as part of this PR? You can add to the Rover section.

Reply:
@MaEtUgR just to be sure, this will be in for 1.15, right? Everything we add in PR's currently that get merged, will be in the next release version?

Reply:
Added release notes.